### PR TITLE
Chinese translation of Performance chapter + some clean ups of previous translations.

### PR DESCRIPTION
**Pull request by chengxicn**
Makes progress on #1067

### CODE REVIEW

File: src/content/zh-CN/2019/performance.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+  <div id="fig1-description" class="visually-hidden">1,000个网站的快速，中速和慢速FCP的分布。快速FCP的分布从100％到0％呈线性关系</div>
+  <figcaption id="fig1-caption">图1.网站的快速，中速和慢速FCP性能分布。</figcaption>
+</figure>
+
+在上面的图1中，您可以看到FCP体验是如何在Web上分布的。这个图表从CrUX数据集里面的数百万个网站之中，将分布缩减到1,000个网站，其中每个垂直切片代表一个站点。该图表按快速FCP的百分比排序，快速FCP体验是1秒之内发生的。慢速的体验会在3秒或更长时间内发生，而中速的体验（以前称为“平均”）介于两者之间。在图表的极端情况下，有些网站拥有近100％的快速体验，而有些网站则具有近100％的慢速体验。在这两者之间，兼具快速，中速和慢速性能的网站似乎更倾向于快速或中速，而不是慢速，这很好。
+
+<p class="note">注意：当用户的性能下降时，很难说出原因是什么。网站本身可能设计得很差且效率低下。或可能存在其他环境因素，例如用户的连接速度慢，没有缓存等。因此，在查看该现场数据时，我们更倾向于说用户自身的体验缓慢，而不一定是网站。</p>
+
+为了对网站是否足够**快速**进行分类，我们将使用新的[PageSpeed Insights](https://developers.google.com/speed/docs/insights/v5/about#categories) (PSI) 方法，其中至少有75％的网站FCP体验必须快于1秒。同样，一个足够**慢速**的网站具有25％或更多的FCP体验慢于3秒。我们说一个网站不符合以上两个条件时，它们的性能就是**中速**。
+
```
Comment (bazzadp) [inline src/content/zh-CN/2019/performance.md:59]: Remove extra line so same line numbering as English

```suggestion
```

File: src/content/zh-CN/2019/performance.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+<figure id="fig2">
+  <a href="/static/images/2019/performance/fig2.png">
+    <img src="/static/images/2019/performance/fig2.png" alt="图2.标有快速，中等或慢速FCP的网站的分布。" aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=36103372&amp;format=interactive">
+  </a>
+  <div id="fig2-description" class="visually-hidden">条形图显示，有13％的网站具有快速FCP，有66％的网站具有中速FCP，有20％的网站具有慢速的FCP。</div>
+  <figcaption id="fig2-caption">图2.标有快速，中速或慢速FCP的网站的分布。</figcaption>
+</figure>
+
+图2的结果表明，只有13％的网站被认为是快速的。这表明仍然有很多改进的空间，但是许多网站都在快速，一致地绘制有意义的内容。三分之二的网站具有中速的FCP性能体验。
+
```
Comment (bazzadp) [inline src/content/zh-CN/2019/performance.md:70]: Remove extra line so same line numbering as English:
```suggestion
```

File: src/content/zh-CN/2019/performance.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+与以前的指标相比，FID总体表现的分布更倾向于快速和中速的体验，而不是慢速的。40％的网站具有快速FID，只有15％的网站具有慢速FID。FID的本质是一种交互性指标-与受网络速度限制的加载指标相反-带来了一种完全不同的性能表现方式。
+
+#### 按设备类型看FID
+
+<figure id="fig14">
+  <a href="/static/images/2019/performance/fig14.png">
+    <img src="/static/images/2019/performance/fig14.png" alt="图14.快速，中度和慢速FID性能'桌面'站点的分布" aria-labelledby="fig14-caption" aria-describedby="fig14-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=16379508&amp;format=interactive">
+  </a>
+  <div id="fig14-description" class="visually-hidden">1,000个桌面网站的快速，中速和慢速FID分布。对于最快的四分之三的网站，快速FID的分布非常缓慢地从100％降低到90％。之后，快速FID会略微降低到75％。几乎所有桌面网站都拥有超过75％的快速FID体验。
+</div>
```
Comment (bazzadp) [inline src/content/zh-CN/2019/performance.md:217]: Remove extra line so same line numbering as English
```suggestion
  <div id="fig14-description" class="visually-hidden">1,000个桌面网站的快速，中速和慢速FID分布。对于最快的四分之三的网站，快速FID的分布非常缓慢地从100％降低到90％。之后，快速FID会略微降低到75％。几乎所有桌面网站都拥有超过75％的快速FID体验。</div>
```

File: src/content/zh-CN/2019/performance.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+  - 平板
+- 移动用户有效链接类型(ECT)
+  - 离线
+  - 缓慢的2G
+  - 2G
+  - 3G
+  - 4G
+- 用户地理位置
+
+用户体验按月量度，包括绘制、加载和交互性指标。我们要看的第一个度量是[首次有内容的绘制](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) (FCP)。这个指标是用户等待在屏幕上显示出有用处的页面内容（例如图像或文本）所花费的时间。然后，我们来看一下加载指标[首包时间](https://developer.mozilla.org/en-US/docs/Glossary/time_to_first_byte) (TTFB)。这是网页从用户导航到接收到响应的第一个字节所花费的时间的指标。最后，我们要看的最后一个真实用户指标是[首次输入延迟](https://developers.google.com/web/updates/2018/05/first-input-delay) (FID)。这是一个相对较新的度量标准，它表示的是用户体验的部分，而不是加载性能。它测量从用户第一次与页面UI交互到浏览器主线程准备好处理事件的时间。
```
Comment (bazzadp) [inline src/content/zh-CN/2019/performance.md:40]: Use local references if they exist:

```suggestion
用户体验按月量度，包括绘制、加载和交互性指标。我们要看的第一个度量是[首次有内容的绘制](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#first_paint_and_first_contentful_paint) (FCP)。这个指标是用户等待在屏幕上显示出有用处的页面内容（例如图像或文本）所花费的时间。然后，我们来看一下加载指标[首包时间](https://developer.mozilla.org/zh-CN/docs/Glossary/%E7%AC%AC%E4%B8%80%E5%AD%97%E8%8A%82%E6%97%B6%E9%97%B4) (TTFB)。这是网页从用户导航到接收到响应的第一个字节所花费的时间的指标。最后，我们要看的最后一个真实用户指标是[首次输入延迟](https://developers.google.com/web/updates/2018/05/first-input-delay) (FID)。这是一个相对较新的度量标准，它表示的是用户体验的部分，而不是加载性能。它测量从用户第一次与页面UI交互到浏览器主线程准备好处理事件的时间。
```

File: src/content/zh-CN/2019/performance.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+<figure id="fig8">
+  <a href="/static/images/2019/performance/nav-timing.png">
+    <img alt="页面导航中Navigation Timing API的事件图" src="/static/images/2019/performance/nav-timing.png" alt="图8.页面导航中Navigation Timing API的事件图。" aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="2580" height="868">
+  </a>
+  <div id="fig8-description" class="visually-hidden">该图展示了页面加载过程的网络阶段：startTime (promptForUnload)，重定向，AppCache，DNS，TCP，请求，响应，处理，以及加载。</div>
+  <figcaption id="fig8-caption">图8.页面导航中Navigation Timing API的事件图。</figcaption>
+</figure>
+
+为了帮助解释TTFB及其影响因素，让我们借鉴一份[Navigation Timing API 规范](https://developer.mozilla.org/en-US/docs/Web/API/Navigation_timing_API)中的图表。在上面的图8中，TTFB是从`开始时间 startTime` 到 `响应时间 responseStart`，包括介于两者之间的所有内容：`卸载 unload`，`重定向 redirects`，`AppCache`，`DNS`，`SSL`，`TCP`，以及服务器处理请求的时间。在这些条件下，我们来看看这一指标用户的体验。
```
Comment (bazzadp) [inline src/content/zh-CN/2019/performance.md:144]: Use local references if they exist:
```suggestion
为了帮助解释TTFB及其影响因素，让我们借鉴一份[Navigation Timing API 规范](https://developer.mozilla.org/zh-CN/docs/Web/API/Navigation_timing_API)中的图表。在上面的图8中，TTFB是从`开始时间 startTime` 到 `响应时间 responseStart`，包括介于两者之间的所有内容：`卸载 unload`，`重定向 redirects`，`AppCache`，`DNS`，`SSL`，`TCP`，以及服务器处理请求的时间。在这些条件下，我们来看看这一指标用户的体验。
```

File: src/templates/zh-CN/2019/featured_chapters.html [hunk 6]
```diff
@@ -80,4 +80,4 @@
   {%- set featured_chapter_quote = "HTTP/2是近20年来对web主要传输协议的第一次重大更新。它带来了丰富的期望:它承诺免费的性能提升而没有缺点。更重要的是，我们可以停止做所有 HTTP/1.1曾因为低效率的缘故，强迫我们叠加的入侵式解决方案。在HTTP/2世界中，绑定、弹跳、内联甚至分片域都将成为反模式，因为默认情况下会提供改进的性能。之前的优化技术Bundling, spriting, inlining,甚至域分片都会在HTTP/2 的世界中成为反模式, 因为它在默认情况就提供改进的性能。本章探讨了这种相对较新的技术在现实世界中的表现。" %}
   {%- set featured_chapter_stats = {"stat1":"95%","label1":"可以使用HTTP/2的用户","stat2":"27.83%","label2":"有HTTP/2优先级问题的请求","stat3":"8.38%","label3":"支持QUIC的站点"} %}
 {%- endif %}
-{{ featuredChapter(featured_chapter, featured_chapter_quote, featured_chapter_stats) }}
+{{ featuredChapter(featured_chapter, featured_chapter_quote, featured_chapter_stats) }}
```
Comment (bazzadp) [inline src/templates/zh-CN/2019/featured_chapters.html:83]: Missing a new line
```suggestion
{{ featuredChapter(featured_chapter, featured_chapter_quote, featured_chapter_stats) }}

```

**Comment by bazzadp** [comment]
Also not sure if you can use the "makes progress on XXX" in title, or only in comments, so moved that to first comment as well.

File: src/content/zh-CN/2019/performance.md (hunk 5 above)
Comment (chengxicn) [reply src/content/zh-CN/2019/performance.md:142]: the local references didn't finish translation yet,so it's better stay in English version.

**Comment by chengxicn** [comment]
I'll try to reset now.

**Comment by bazzadp** [comment]
> I'll try to reset now.

Don't reset now or you'll throw away all your hard work here! Need to wait until after this is merged.

**Comment by chengxicn** [comment]
Thank you! @bazzadp I'll wait the merge.
and I'll try to not use main, was focusing on translation, but didn't fully understand git, need some time to learn the main/branch logic.

**Comment by bazzadp** [comment]
No worries - do read that wiki of mine: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide

It's long, but hopefully makes all this clearer! If not let me know what bits are unclear and will try to improve.

**Comment by bazzadp** [comment]
Gonna do a release so will merge this now. @limhenry if you get a chance to review at a later stage feel free to comment here or open a new PR with suggested changes.

**Pull request merged**